modin/engines/ray/pandas_on_ray/frame/partition.py

```diff
@@ -20,11 +20,18 @@
 import ray
 from ray.worker import RayTaskError
 from ray.services import get_node_ip_address
+from packaging import version
+ObjectIDType = ray.ObjectRef
```

Comment:
Is this a good name @devin-petersohn, or would you prefer something like `object_id_cls`?